### PR TITLE
Add F411RX Target and cleaned unused declaration 

### PR DESCRIPTION
**Pull request by Si-iF**
Add F411RX Target ( IFRC-IFLIGHT_BLITZ_F411RX.config ) and cleaned unused delcaration such as NONE or AUTO strings other two targets (IFRC-IFLIGHT_F411_PRO.config , IFRC-IFLIGHT_SUCCEX_E_F4.config )

### CODE REVIEW

**Comment by Asizon** [comment]
Duplicated PR https://github.com/betaflight/unified-targets/pull/533#event-5725203142
Please dont close and reopen another PR.

**Comment by Si-iF** [comment]
> Duplicated PR [#533 (comment)](https://github.com/betaflight/unified-targets/pull/533#event-5725203142) Please dont close and reopen another PR.

I'm very sorry @Asizon , I made a mess in the previous PR that i could not resolve so I closed and opened with all correct commits and files. Very sorry for what happened it will never happen again

**Comment by Asizon** [comment]
You need to leard some Git comands before push a pull request, Squashing and rebasing commits is a very normal action here.

**Comment by Si-iF** [comment]
@Asizon I will next time thanks!

File: configs/default/IFRC-IFLIGHT_BLITZ_F411RX.config [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin A10 0
+# pin A10: DMA2 Stream 6 Channel 0
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
```
Comment (DusKing1) [inline configs/default/IFRC-IFLIGHT_BLITZ_F411RX.config:69]: If this is a board with built-in RX why you enable `RX_SERIAL` instead of `RX_SPI`?

**Comment by DusKing1** [comment]
This is a duplicated PR, and #533 has already become an orphan.

**Comment by Si-iF** [comment]
@haslinghuis @mikeller Sorry for disturbing you guys, can we please merge this target ? We also have another target that we have to upload and we are waiting for this merge. Sorry for the mess we created with the previous PR we closed.

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**